src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/CodeStyle/FadingOptions.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.CodeStyle
+{
+    internal static class FadingOptions
```

Comment:
>FadingOptions [](http://example.com/codeflow?start=26&length=13)

Rename to `FadingOptionsStorage` as this type defines storage for options, not the actual option values.